src/validateInput.js

```diff
@@ -8,7 +8,7 @@ const CHARACTERS = '\\p{L}\\p{M}'
 const INPUT_BASE_REG_EXP = new RegExp(`-0-9.,;:<>?/\n${CHARACTERS}'_ &()/+@%`, 'u')
 const DIGITS = '0-9'
 
-const NUMBER = /^-?\d*[\.]?\d+$/
+const NUMBER = /^-?\d*[.]?\d+$/
```

Comment:
these mean different things. i think the intention for the initial change is to explicitly match dot character for decimals.

Reply:
Dot character in the set do not need to be escaped by backslash so it is interpreted as the pure dot character

Reply:
Agree with @dictor93